wiki_articles/vscode.md

```diff
@@ -0,0 +1,33 @@
+# Getting Started With VS Code for C/C++
+
+## DISCLAIMER FOR BEGINNERS
+Do not use VS Code.
+It's better to use an IDE that works out of the box:
+- [Visual Studio Community](https://visualstudio.microsoft.com/vs/community/) for Windows (free)
+- [CLion](https://www.jetbrains.com/clion/) on any platform (free for students)
+- [XCode](https://developer.apple.com/xcode/) on macOS (free)
+- [repl.it](https://replit.com/site/ide) online IDE (free for public projects)
+
+## 1. Install The C/C++ Extension by Microsoft
+Install [the extension](https://code.visualstudio.com/docs/languages/cpp) and
+follow the tutorial.
+Don't use third-party plugins like *Code Runner*, use the official stuff.
+
+## 2. Install A Compiler (possibly GCC through MinGW, or GCC through WSL)
+You still need to install a compiler, which is the thing that turns your code
+into an executable, separately.
+Consider using
+[Windows Subsystem for Linux](https://code.visualstudio.com/docs/cpp/config-wsl)
+instead of MinGW, since Linux makes development much easier.
+
+## 3. Install and Set Up CMake for VSCode
+VSCode lets you configure your project through `build.json` and `launch.json`
+config files.
+However, it's *much easier* and *more portable* if you use
```

Comment:
I think it'd be better to not mention launch.json or tasks.json at all and just give instructions for getting started with cmake

Reply:
I wouldn't want to omit these completely. It's still relevant to know them, since a lot of tutorials will include these. There are also people who don't want to use CMake, or aren't allowed/meant to because of their assignments.